app/Http/Controllers/MetaDataController.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\MetaData;
+use Illuminate\Http\Request;
+
+class MetaDataController extends Controller
+{
+    // Get metadata
+    public function getMetaData($url) {
+        return DB::table('meta_data')->where('url', $url)->last();
```

Comment:
Is er een reden om hier geen eloquent model voor te gebruiken?

Reply:
nee